draft-iab-wire-image.md

```diff
@@ -61,53 +61,69 @@ discuss in this document. It is the wire image, not the protocol's
 specification, that determines how third parties on the network paths among
 protocol participants will interact with that protocol.
 
-Several documents currently under discussion in IETF working groups and the
-IETF in general, for example
-{{?QUIC-MANAGEABILITY=I-D.ietf-quic-manageability}},
-{{?EFFECT-ENCRYPT=I-D.mm-wg-effect-encrypt}}, and
-{{?TRANSPORT-ENCRYPT=I-D.fairhurst-tsvwg-transport-encrypt}}, discuss in
-part impacts on the third-party use of wire images caused by a migration from
-protocols whose wire images are largely not confidentiality protected (e.g.
-HTTP over TCP) to protocols whose wire images are confidentiality protected
-(e.g. H2 over QUIC).
-
-This document presents the wire image abstraction with the hope that it can
-shed some light on these discussions.
+The increasing deployment of transport-layer security {{?RFC8226}} to protect
+application-layer headers and payload, as well as the definition and deployment
+of QUIC {{?I-D.ietf-quic-transport}}, a transport protocol which encrypts most
+of its own control information, bring new relevance to this question. QUIC is,
+in effect, the first IETF-defined transport protocol to take care of the
+minimization of its own wire image, for reasons of ossification prevention as
+well as reducing information radiation off the protocol for end-to-end privacy.
```

Comment:
This last sentence seems a bit wordy.

Perhaps:
`QUIC is, in effect, the first IETF-defined transport protocol to take care of the minimization of its own wire image, to prevent ossification and improve end-to-end privacy by reducing information radiation.`

Reply:
yep done, thx.